tests/test_incorrect_base_class.py

```diff
@@ -0,0 +1,54 @@
+"""Tests for incorrect base class checker."""
+from pathlib import Path
+
+import astroid
+from pylint.testutils import CheckerTestCase
+from pylint.testutils import MessageTest
+from pytest import mark
+
+from pylint_nautobot.incorrect_base_class import NautobotIncorrectBaseClassChecker
+
+from .utils import assert_no_message
+
+_INPUTS_PATH = Path(__file__).parent / "inputs/incorrect-base-class"
```

Comment:
Sorry that I didn't see this before, but for this can we use the newer pattern utilizing [this](https://docs.python.org/3/library/importlib.resources.html#importlib.resources.files)? The backport library for this std library module `importlib_resources` is already a dependency as @cmsirbu used it [here](https://github.com/nautobot/pylint-nautobot/blob/24a79be6e09a0b054869e7b16e4c5b78eca67efa/pylint_nautobot/utils.py#L8).

Reply:
NP, willing to make code better, and I like this feature. :-) However, in this case, I'm not sure, whether using resources is appropriate, as `tests` dir is not part of the package. As far as I understand it, `importlib.resources`: ` provide access to resources within packages`. It should be used to read resources inside `pylint_nautobot` dir, but not in `tests`. WDYT? 

https://docs.python.org/3/library/importlib.resources.html

Reply:
Oh yeah, good point. I tried finding something better but no luck.. I guess we stay with this approach for now.